dbt/adapters/databricks/connections.py

```diff
@@ -138,11 +156,18 @@ def __post_init__(self) -> None:
         self.connection_parameters = connection_parameters
```

Comment:
can you add 

```python
self.config = databricks.sdk.core.Config(
            host=self.host,
            token=self.token,
            client_id=self.client_id,
            client_secret=self.client_secret,
            azure_client_id=self.client_id, # or self.azure_client_id
            azure_client_secret=self.client_secret, # or self.azure_client_secret
            azure_tenant_id=self.azure_tenant_id, # we need azure tenant ID for AAD SPN auth,
            # this will open the browser if we don't have the token or client_secret set. try working with `databricks-cli` as client id for now. 
            auth_type='external-browser' if (not self.token and not self.client_secret) else None,
     )
```

Reply:
Also, are you sure you want to skip loading info from standard environment variables?

Reply:
See https://github.com/databricks/databricks-sdk-py/blob/main/databricks/sdk/core.py#L113-L131 for details